src/main/java/com/denizenscript/ddiscordbot/properties/DiscordElementTagExtensions.java

```diff
@@ -0,0 +1,28 @@
+package com.denizenscript.ddiscordbot.properties;
+
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.core.TimeTag;
+import net.dv8tion.jda.api.utils.TimeUtil;
+
+public class DiscordElementTagExtensions {
+
+    public static void register() {
+
+        // <--[tag]
+        // @attribute <ElementTag.discord_id_to_time>
+        // @returns TimeTag
+        // @group math
+        // @description
+        // Each discord ID (for channels, users, guilds, ect) stores the exact time it was created.

```

Comment:
`etc.`

Reply:
It's not _stored_ - it _is_ a timestamp. ID's are just unix timestamps with some arcane discordmath applied.